scripts/create-yearly-es-globals.mjs

```diff
@@ -0,0 +1,21 @@
+import fs from 'node:fs';
+
+const year = new Date().getFullYear();
+const dataFile = new URL(`../data/es${year + 1}.mjs`, import.meta.url);
+const workflowFile = new URL('../.github/workflows/es-yearly-globals.yml', import.meta.url);
+
+fs.writeFileSync(
+	dataFile,
+	`export {default} from './es${year}.mjs';\n`,
+);
+
+fs.writeFileSync(
+	workflowFile,
+	fs.readFileSync(workflowFile, 'utf8')
+		.replaceAll(
+			`Add \`es${year + 1}\` globals`,
+			`Add \`es${year + 2}\` globals`,
```

Comment:
In 2025, this will update title from ``Add `es2026` globals`` to ``Add `es2027` globals``